lib/config/migrations/custom/enabled-managers-migration.ts

```diff
@@ -1,14 +1,20 @@
 import is from '@sindresorhus/is';
 import { AbstractMigration } from '../base/abstract-migration';
+import { isCustomManager } from '../../../modules/manager/custom';
 
 export class EnabledManagersMigration extends AbstractMigration {
   override readonly propertyName = 'enabledManagers';
 
   override run(value: unknown): void {
-    if (is.array(value)) {
-      const newValue = value.map((manager) =>
-        manager === 'yarn' ? 'npm' : manager
-      );
+    if (is.array<string>(value, is.string)) {
```

Comment:
Use early return

Reply:
Didn't get this

Reply:
This is a pattern to reduce the number of nested blocks. It improves the readability. 
https://gomakethings.com/the-early-return-pattern-in-javascript/

Reply:
e.g. "if not array then return"